web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx

```diff
@@ -265,7 +274,33 @@ export const AllProjects: React.FC<React.PropsWithChildren<unknown>> = () => {
               purchaseDetailsTitles={purchaseDetailsTitles}
               buttons={buttons}
               useProjectCardButton={IS_REGEN}
-              hasBottomCard={IS_REGEN}
+              creditIcon={
+                <>
```

Comment:
We need a conditional to only show this for Terrasos, right now it's displaying on the Regen client too.